packages/react-examples/src/react/FocusTrapZone/FocusTrapZone.Box.Example.tsx

```diff
@@ -50,3 +62,24 @@ export const FocusTrapZoneBoxExample: React.FunctionComponent = () => {
     </Stack>
   );
 };
+
+const IFrameWrapper = (props: { onClick: () => void }): JSX.Element => {
+  const { onClick } = props;
+
+  const iframeRef = React.useRef<HTMLIFrameElement | null>(null);
+
+  React.useEffect(() => {
+    if (iframeRef.current) {
+      const contentWindow = iframeRef.current.contentWindow;
+
+      if (contentWindow) {
+        const root = contentWindow.document.createElement('div');
+        contentWindow.document.body.appendChild(root);
+
+        ReactDOM.render(<DefaultButton onClick={onClick}>Button in IFrame</DefaultButton>, root);
+      }
+    }
+  }, []);
```

Comment:
React Hook React.useEffect has a missing dependency: 'onClick'. 
Either include it or remove the dependency array